### PR TITLE
added display name to user profile

### PR DESCRIPTION
**Pull request by yanchen01**

### CODE REVIEW

File: mobile/flutter_app/lib/widgets/dashboard/dashboardTop.dart [hunk 1]
```diff
@@ -1,65 +1,75 @@
...
+                      fontSize: 18,
+                      fontWeight: FontWeight.w500,
+                      decoration: TextDecoration.none),
+                ),
+              ),
+            ),
+          ],
+        ),
+        height: 250);
+  }
```
Comment (myliang47) [inline mobile/flutter_app/lib/widgets/dashboard/dashboardTop.dart:74]: Could you summarize what changes you made here please?

Comment (yanchen01) [reply mobile/flutter_app/lib/widgets/dashboard/dashboardTop.dart:74]: I think it's just my code formatter formatting the spaces... i didn't change anything i believe 😅

Comment (yanchen01) [reply mobile/flutter_app/lib/widgets/dashboard/dashboardTop.dart:74]: Oh, added the gesture signout on the welcome text...

Comment (myliang47) [reply mobile/flutter_app/lib/widgets/dashboard/dashboardTop.dart:74]: Could you add a comment to take out the gesturehandler once we add in a logout button?
// TODO: Take out GestureHandler after logout button is added in

Comment (yanchen01) [reply mobile/flutter_app/lib/widgets/dashboard/dashboardTop.dart]: Done

**Review by myliang47 — APPROVED** [review]
